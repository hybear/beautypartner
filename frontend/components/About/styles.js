import styled from 'styled-components';

export const Content = styled.main`
  display: grid;
  grid-template:
    'profile artwork' auto
    'about artwork' 10vh
    'design development' 60vh / 60% 40%;
  padding: 40px;
`;

export const Profile = styled.div`
  grid-area: profile;
  display: grid;
  align-items: center;
  grid-template-columns: 25% 75%;
  border-bottom: solid 1px ${({ theme }) => theme.color.lightGrey};
  padding-bottom: 40px;
  .picture {
    margin: 0 auto;
    max-height: 180px;
    max-width: 180px;
    border-radius: 100px;
    object-fit: cover;
  }
  .content {
    margin-left: 20px;
    &__sub {
      font-size: 1em;
      i {
        color: ${({ theme }) => theme.color.primary};
      }
    }
    &__name {
      font-size: 1.875em;
      color: ${({ theme }) => theme.color.primary};
      font-family: 'QuadraSans-Bold', sans-serif;
      margin: 5px 0 10px;
      small {
        font-size: 0.5em;
        font-family: 'QuadraSans-Regular', sans-serif;
      }
    }
    &__shortinfo {
      font-size: 1em;
    }
    &__social {
    }
  }
`;

export const Artwork = styled.div``;
export const About = styled.div``;
export const Design = styled.div``;
export const Development = styled.div``;
