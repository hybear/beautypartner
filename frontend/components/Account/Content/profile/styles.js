import styled from 'styled-components';
import { Text } from '../../../General';
import { Button as GButton } from '../../../General/Form';

export const ProfileContainer = styled.div`
  align-items: start;
  display: grid;
  grid-template:
    'title title' auto
    'profile password' auto
    / 60% 1fr;
  grid-gap: 0 20px;
`;

export const Profile = styled.div`
  background: ${({ theme }) => theme.color.lightGrey};
  border-radius: 0.2rem;
  color: ${({ theme }) => theme.color.grey};
  box-shadow: inset 0px 1px 5px 0px rgba(0, 0, 0, 0.15);
  display: grid;
  grid-area: profile;
  ${props => !props.handleProfile && `grid-template-columns: repeat(auto-fill, minmax(calc(100% / 3), 1fr))`};
  grid-gap: 5px;
  /* margin: 10px 0; */
  padding: 20px;
  position: relative;
  p {
    font-family: 'QuadraSans-Bold';
  }
`;

export const ProfileItem = styled.span`
  margin: 5px;
  p {
    margin-bottom: 5px;
    font-size: 1.2em;
  }
`;

export const Password = styled.div`
  form {
    align-items: start;
    display: grid;
    grid-template: 'email email' auto;
    grid-gap: 10px;
  }
`;

export const EmailReset = styled(Text)`
  grid-area: email;
  color: ${({ theme, success }) => success && theme.color.success};
  b {
    font-family: 'QuadraSans-Bold';
  }
`;

export const PasswordContainer = styled.div`
  grid-area: password;
  max-width: 30rem;
  padding: 20px;
  /* margin: 10px 0; */
  border-radius: 0.2rem;
  background: ${({ theme }) => theme.color.lightGrey};
  box-shadow: inset 0px 1px 5px 0px rgba(0, 0, 0, 0.15);
  color: ${({ theme }) => theme.color.grey};
  position: relative;
  > p {
    font-family: 'QuadraSans-Bold';
    font-size: 1.2em;
  }
`;

export const Button = styled(GButton)`
  ${({ profile }) => profile && `margin: 20px 10px 0 0;`}
`;
