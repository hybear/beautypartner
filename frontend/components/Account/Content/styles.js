import styled from 'styled-components';
import { Title as GTitle } from '../../General';

export const Content = styled.div`
  grid-area: cont;
  margin: 20px 0 0 40px;
  padding: 0 20px 0;
  overflow: auto;
  &::-webkit-scrollbar-track {
    box-shadow: inset 0 0 1px rgba(0, 0, 0, 0.3);
    border-radius: 10px;
    background-color: ${({ theme }) => theme.color.white};
  }

  &::-webkit-scrollbar {
    width: 0.5em;
    background-color: ${({ theme }) => theme.color.white};
  }

  &::-webkit-scrollbar-thumb {
    border-radius: 10px;
    background-color: ${({ theme }) => theme.color.lightGrey};
  }

  ${({ theme }) => theme.media.tablet`
      margin: 20px 0 80px;
  `}
`;

export const Title = styled(GTitle)`
  text-align: initial;
  font-size: 1.875em;
  font-family: 'QuadraSans-Bold';
  color: ${({ theme }) => theme.color.grey};
  margin-bottom: 20px;
`;
