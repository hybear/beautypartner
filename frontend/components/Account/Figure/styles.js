import styled, { keyframes } from 'styled-components';

export const FigureContainer = styled.div`
  grid-area: figures;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100%;
  width: 100%;
  ${({ theme }) => theme.media.tablet`
    display: none;
  `}
`;

const bright = keyframes`
  0% {
    filter: contrast(80%);
  }

  50% {
    filter: contrast(120%);
  }

  100% {
    filter: contrast(80%);
  }
`;

export const Image = styled.img`
  mask-image: url('/assets/icon/MaskAccount.svg');
  mask-position: right;
  mask-repeat: no-repeat;
  mask-size: cover;
  object-fit: cover;
  object-position: right;
  height: 100%;
  width: 100%;
  animation: ${bright} 3s linear infinite;
`;
