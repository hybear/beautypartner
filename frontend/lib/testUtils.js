import casual from 'casual';

// seed it so we get consistent results
casual.seed(777);

const fakeItem = () => ({
  __typename: 'Item',
  id: 'abc123',
  listPrice: 180000,
  bestPrice: 155200,
  category: 100,
  reference: '15523',
  title: 'LA MER The Moisturizing Soft Cream',
  description:
    'An ultra-soft face moisturizer that nurtures for beautiful, glowing radiance, leaving skin looking naturally brightened, hydrated, and restored.',
  image: 'laMer.jpg',
  largeImage: 'largelaMer.jpg',
});

const fakeUser = () => ({
  __typename: 'User',
  id: '4234',
  name: casual.name,
  email: casual.email,
  permissions: ['ADMIN'],
  birthday: '1996-03-08T14:18:00.000Z',
  creditCards: [],
  phone: casual.phone,
  balance: 1800,
  orders: [],
  cart: [],
});

const fakeOrderItem = () => ({
  __typename: 'OrderItem',
  id: casual.uuid,
  image: `${casual.word}.jpg`,
  title: casual.words(),
  listPrice: 4234,
  bestPrice: 4234,
  quantity: 1,
  description: casual.words(),
});

const fakeOrder = () => ({
  __typename: 'Order',
  id: 'ord123',
  charge: 'ch_123',
  total: 400,
  cashback: 5,
  paymentMethod: 'stripe',
  items: [fakeOrderItem(), fakeOrderItem()],
  status: ['APROVED'],
  createdAt: '2020-02-17T01:52:31.269Z',
  user: fakeUser(),
});

const fakeCartItem = overrides => ({
  __typename: 'CartItem',
  id: 'omg123',
  quantity: 3,
  item: fakeItem(),
  user: fakeUser(),
  ...overrides,
});

// Fake LocalStorage
class LocalStorageMock {
  constructor() {
    this.store = {};
  }

  clear() {
    this.store = {};
  }

  getItem(key) {
    return this.store[key] || null;
  }

  setItem(key, value) {
    this.store[key] = value.toString();
  }

  removeItem(key) {
    delete this.store[key];
  }
}

export { LocalStorageMock, fakeItem, fakeUser, fakeCartItem, fakeOrder, fakeOrderItem };
