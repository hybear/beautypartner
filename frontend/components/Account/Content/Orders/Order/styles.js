import styled from 'styled-components';
import { Title } from '../../styles';
import { Button } from '../../../../General/Form';

export const OrderStylesContainer = styled.div`
  height: 100%;
  overflow: hidden;
`;

export const OrderTotal = styled.p`
  font-size: 1.1em;
  margin: 0 0 15px;
`;
export const OrderListTitle = styled.div`
  /* display: grid;
  grid-template-columns: repeat(5, auto);
  text-align: center; */
`;
export const OrderStyles = styled.li`
  margin: 10px 0;
  display: block;
  .order {
    display: grid;
    justify-content: space-between;
    align-items: center;
    grid-template-columns: repeat(4, auto);
    padding: 20px;
    transition: all 0.2s;
    text-align: center;
    color: ${({ theme }) => theme.color.grey};
    text-decoration: none;
    background: ${({ theme }) => theme.color.lightGrey};
    box-shadow: inset 0px 1px 5px 0px rgba(0, 0, 0, 0.15);
    border-radius: 0.2rem;

    .order__title {
      font-family: 'QuadraSans-Bold', sans-serif;
      margin-bottom: 10px;
      font-size: 1.25em;
    }

    .order__info {
      &.cashback {
        small {
          display: block;
          background: ${({ theme }) => theme.color.primary};
          color: ${({ theme }) => theme.color.white};
          padding: 0.1em 0;
          border-radius: 2px;
        }
      }
    }
  }
`;

export const OrderButton = styled(Button)`
  a {
    color: ${({ theme }) => theme.color.white};
    &:hover {
      text-decoration: none;
    }
  }
`;

export const OrderList = styled.ul`
  overflow-x: hidden;
  overflow-y: auto;
  margin-bottom: 20px;
  &::-webkit-scrollbar-track {
    box-shadow: inset 0 0 1px rgba(0, 0, 0, 0.3);
    border-radius: 10px;
    background-color: #f5f5f5;
  }

  &::-webkit-scrollbar {
    width: 0.5em;
    background-color: #f5f5f5;
  }

  &::-webkit-scrollbar-thumb {
    border-radius: 10px;
    background-color: ${({ theme }) => theme.grey};
  }
`;
