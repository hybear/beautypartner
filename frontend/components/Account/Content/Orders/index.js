import { Query } from 'react-apollo';
import gql from 'graphql-tag';
import formatMoney from '../../../../lib/formatMoney';
import { format, formatDistance, parseISO } from 'date-fns';
import Link from 'next/link';

// UI
import { Title } from '../styles'; // Content
import {
  OrderStylesContainer,
  OrderListTitle,
  OrderList,
  OrderStyles,
  OrderTitle,
  OrderTotal,
  OrderButton,
} from './styles';

// ANIMATIONS
import Loading from '../../../General/Animations/ContentLoading';

const ALL_ORDERS_QUERY = gql`
  query ALL_ORDERS_QUERY {
    orders(orderBy: createdAt_DESC) {
      id
      charge
      total
      createdAt
      status
      paymentMethod
      cashback
      user {
        id
      }
      items {
        id
        title
        description
        image
        listPrice
        bestPrice
        quantity
      }
    }
  }
`;

const Order = order => (
  <OrderStyles>
    <Link href={`/orders/${order.id}`}>
      <a className="order">
        <span className="order__title idOrders">Order ID</span>
        <span className="order__title statusOrders">Status</span>
        <span className="order__title dateOrders">Date</span>
        <span className="order__title totalOrders">Total</span>
        <span className="order__title cashbackOrders">Cashback</span>
        <span className="order__info idValOrders">{order.id}</span>
        <span className="order__info statusValOrders">{order.status}</span>
        <span className="order__info dateValOrders">{formatDistance(parseISO(order.createdAt), new Date())} ago</span>
        <span className="order__info totalValOrders">{formatMoney(order.total)}</span>
        <span className="order__info cashbackValOrders">
          +
          {formatMoney(
            (order.items.reduce((tally, cartItem) => tally + cartItem.bestPrice * cartItem.quantity, 0) *
              order.cashback) /
              100
          )}
          <small>{order.cashback}%</small>
        </span>
      </a>
    </Link>
  </OrderStyles>
);

const Orders = () => {
  return (
    <Query query={ALL_ORDERS_QUERY}>
      {({ data, error, loading }) => {
        if (loading) return <Loading isStoped={false} />;
        if (error) return <Error error={error} />;

        const orders = data.orders;
        return (
          <OrderStylesContainer data-test="orders">
            <Title>Orders</Title>
            <OrderTotal>You have a total of {orders.length} orders</OrderTotal>
            <OrderListTitle></OrderListTitle>
            <OrderList className="orderList">
              {orders.map((order, i) => (
                <Order {...order} key={i} />
              ))}
            </OrderList>
            <OrderButton primary>
              <Link href="/products">
                <a>New Order</a>
              </Link>
            </OrderButton>
          </OrderStylesContainer>
        );
      }}
    </Query>
  );
};

export default Orders;
export { ALL_ORDERS_QUERY };
