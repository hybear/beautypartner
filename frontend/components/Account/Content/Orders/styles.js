import styled from 'styled-components';
import { Title } from '../styles';

export const OrderStylesContainer = styled.div`
  height: 100%;
  overflow: hidden;
`;

export const OrderTotal = styled.p`
  font-size: 1.1em;
  margin: 0 15px 20px;
`;

export const OrderList = styled.ul`
  height: 100%;
  overflow-x: hidden;
  overflow-y: auto;
  padding-bottom: 40px;
  &::-webkit-scrollbar-track {
    box-shadow: inset 0 0 1px rgba(0, 0, 0, 0.3);
    border-radius: 10px;
    background-color: #f5f5f5;
  }

  &::-webkit-scrollbar {
    width: 0.5em;
    background-color: #f5f5f5;
  }

  &::-webkit-scrollbar-thumb {
    border-radius: 10px;
    background-color: ${({ theme }) => theme.grey};
  }
`;
