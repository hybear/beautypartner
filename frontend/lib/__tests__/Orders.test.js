import Orders, { ALL_ORDERS_QUERY } from '@components/Account/Content/Orders';
import { MockedProvider } from '@apollo/react-testing';
import { act } from 'react-dom/test-utils';
import wait from 'waait';
import { fakeOrder } from '../testUtils';
import toJson from 'enzyme-to-json';
import { ThemeProviderWrapper } from '../../styles/theme';

const mocks = [
  {
    request: { query: ALL_ORDERS_QUERY },
    result: {
      data: {
        orders: [fakeOrder()],
      },
    },
  },
];

describe('<Order/>', () => {
  it('render orders and display one order', async () => {
    await act(async () => {
      const wrapper = mount(
        <MockedProvider mocks={mocks}>
          <Orders />
        </MockedProvider>,
        {
          wrappingComponent: ThemeProviderWrapper,
        }
      );

      await wait(0);
      wrapper.update();

      expect(toJson(wrapper.find('.orderList ul')).children.length).toBe(1);
      expect(toJson(wrapper.find('.orderList ul')).children[0].type).toBe('Order');
    });
  });

  it('matches snapshot', async () => {
    await act(async () => {
      const wrapper = mount(
        <MockedProvider mocks={mocks}>
          <Orders />
        </MockedProvider>,
        {
          wrappingComponent: ThemeProviderWrapper,
        }
      );

      await wait(0);
      wrapper.update();

      expect(wrapper.find('[data-test="orders"]')).toMatchSnapshot();
    });
  });
});
