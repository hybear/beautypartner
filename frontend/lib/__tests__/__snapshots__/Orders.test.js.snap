// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Order/> matches snapshot 1`] = `
Array [
  <styles__OrderStylesContainer
    data-test="orders"
  >
    <div
      className="styles__OrderStylesContainer-sc-1jlrbgn-0 cBYkdr"
      data-test="orders"
    >
      <styles__Title>
        <h2
          className="Title-qmiaj1-0 styles__Title-sc-1ht2lvb-1 kjYrfC"
        >
          Orders
        </h2>
      </styles__Title>
      <styles__OrderTotal>
        <p
          className="styles__OrderTotal-sc-1jlrbgn-1 imIAYO"
        >
          You have a total of 
          1
           orders
        </p>
      </styles__OrderTotal>
      <styles__OrderListTitle>
        <div
          className="styles__OrderListTitle-sc-1jlrbgn-2 hZDYPi"
        />
      </styles__OrderListTitle>
      <styles__OrderList
        className="orderList"
      >
        <ul
          className="styles__OrderList-sc-1jlrbgn-5 hYEEes orderList"
        >
          <Order
            __typename="Order"
            cashback={5}
            charge="ch_123"
            createdAt="2020-02-17T01:52:31.269Z"
            id="ord123"
            items={
              Array [
                Object {
                  "__typename": "OrderItem",
                  "bestPrice": 4234,
                  "description": "et modi tenetur amet modi reprehenderit omnis",
                  "id": "3a430a73-c9e9-4bcc-b46a-41618965ffea",
                  "image": "reprehenderit.jpg",
                  "listPrice": 4234,
                  "quantity": 1,
                  "title": "soluta non omnis consequatur enim quia autem",
                },
                Object {
                  "__typename": "OrderItem",
                  "bestPrice": 4234,
                  "description": "sapiente laudantium molestias assumenda quasi adipisci mollitia",
                  "id": "2393e090-592f-4d03-b808-c9d81098deec",
                  "image": "sint.jpg",
                  "listPrice": 4234,
                  "quantity": 1,
                  "title": "quia sed exercitationem omnis laborum exercitationem est",
                },
              ]
            }
            key="0"
            paymentMethod="stripe"
            status={
              Array [
                "APROVED",
              ]
            }
            total={400}
            user={
              Object {
                "__typename": "User",
                "id": "4234",
              }
            }
          >
            <styles__OrderStyles>
              <li
                className="styles__OrderStyles-sc-1jlrbgn-3 fdmfdY"
              >
                <Link
                  href="/orders/ord123"
                >
                  <a
                    className="order"
                    href="/orders/ord123"
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                  >
                    <span
                      className="order__title idOrders"
                    >
                      Order ID
                    </span>
                    <span
                      className="order__title statusOrders"
                    >
                      Status
                    </span>
                    <span
                      className="order__title dateOrders"
                    >
                      Date
                    </span>
                    <span
                      className="order__title totalOrders"
                    >
                      Total
                    </span>
                    <span
                      className="order__title cashbackOrders"
                    >
                      Cashback
                    </span>
                    <span
                      className="order__info idValOrders"
                    >
                      ord123
                    </span>
                    <span
                      className="order__info statusValOrders"
                    >
                      APROVED
                    </span>
                    <span
                      className="order__info dateValOrders"
                    >
                      11 days
                       ago
                    </span>
                    <span
                      className="order__info totalValOrders"
                    >
                      $
                      400.00
                    </span>
                    <span
                      className="order__info cashbackValOrders"
                    >
                      +
                      $4.23
                      <small>
                        5
                        %
                      </small>
                    </span>
                  </a>
                </Link>
              </li>
            </styles__OrderStyles>
          </Order>
        </ul>
      </styles__OrderList>
      <styles__OrderButton
        primary={true}
      >
        <button
          className="Button-sc-1hbypwi-0 styles__OrderButton-sc-1jlrbgn-4 kPSbpN"
        >
          <Link
            href="/products"
          >
            <a
              href="/products"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              New Order
            </a>
          </Link>
        </button>
      </styles__OrderButton>
    </div>
  </styles__OrderStylesContainer>,
  <div
    className="styles__OrderStylesContainer-sc-1jlrbgn-0 cBYkdr"
    data-test="orders"
  >
    <styles__Title>
      <h2
        className="Title-qmiaj1-0 styles__Title-sc-1ht2lvb-1 kjYrfC"
      >
        Orders
      </h2>
    </styles__Title>
    <styles__OrderTotal>
      <p
        className="styles__OrderTotal-sc-1jlrbgn-1 imIAYO"
      >
        You have a total of 
        1
         orders
      </p>
    </styles__OrderTotal>
    <styles__OrderListTitle>
      <div
        className="styles__OrderListTitle-sc-1jlrbgn-2 hZDYPi"
      />
    </styles__OrderListTitle>
    <styles__OrderList
      className="orderList"
    >
      <ul
        className="styles__OrderList-sc-1jlrbgn-5 hYEEes orderList"
      >
        <Order
          __typename="Order"
          cashback={5}
          charge="ch_123"
          createdAt="2020-02-17T01:52:31.269Z"
          id="ord123"
          items={
            Array [
              Object {
                "__typename": "OrderItem",
                "bestPrice": 4234,
                "description": "et modi tenetur amet modi reprehenderit omnis",
                "id": "3a430a73-c9e9-4bcc-b46a-41618965ffea",
                "image": "reprehenderit.jpg",
                "listPrice": 4234,
                "quantity": 1,
                "title": "soluta non omnis consequatur enim quia autem",
              },
              Object {
                "__typename": "OrderItem",
                "bestPrice": 4234,
                "description": "sapiente laudantium molestias assumenda quasi adipisci mollitia",
                "id": "2393e090-592f-4d03-b808-c9d81098deec",
                "image": "sint.jpg",
                "listPrice": 4234,
                "quantity": 1,
                "title": "quia sed exercitationem omnis laborum exercitationem est",
              },
            ]
          }
          key="0"
          paymentMethod="stripe"
          status={
            Array [
              "APROVED",
            ]
          }
          total={400}
          user={
            Object {
              "__typename": "User",
              "id": "4234",
            }
          }
        >
          <styles__OrderStyles>
            <li
              className="styles__OrderStyles-sc-1jlrbgn-3 fdmfdY"
            >
              <Link
                href="/orders/ord123"
              >
                <a
                  className="order"
                  href="/orders/ord123"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                >
                  <span
                    className="order__title idOrders"
                  >
                    Order ID
                  </span>
                  <span
                    className="order__title statusOrders"
                  >
                    Status
                  </span>
                  <span
                    className="order__title dateOrders"
                  >
                    Date
                  </span>
                  <span
                    className="order__title totalOrders"
                  >
                    Total
                  </span>
                  <span
                    className="order__title cashbackOrders"
                  >
                    Cashback
                  </span>
                  <span
                    className="order__info idValOrders"
                  >
                    ord123
                  </span>
                  <span
                    className="order__info statusValOrders"
                  >
                    APROVED
                  </span>
                  <span
                    className="order__info dateValOrders"
                  >
                    11 days
                     ago
                  </span>
                  <span
                    className="order__info totalValOrders"
                  >
                    $
                    400.00
                  </span>
                  <span
                    className="order__info cashbackValOrders"
                  >
                    +
                    $4.23
                    <small>
                      5
                      %
                    </small>
                  </span>
                </a>
              </Link>
            </li>
          </styles__OrderStyles>
        </Order>
      </ul>
    </styles__OrderList>
    <styles__OrderButton
      primary={true}
    >
      <button
        className="Button-sc-1hbypwi-0 styles__OrderButton-sc-1jlrbgn-4 kPSbpN"
      >
        <Link
          href="/products"
        >
          <a
            href="/products"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            New Order
          </a>
        </Link>
      </button>
    </styles__OrderButton>
  </div>,
]
`;
